### DISCUSSION TITLE
nostd support

### DISCUSSION THREAD

**Discussion by abergmeier** [slint-ui/slint#6755] [General] [answered]
Is there any `nostd` for slint?

**Comment by ogoffart** [comment, marked as answer]
Yes. There is a enabled by default `std` feature in the `slint` crate. If you don't enable it, you are nostd.

See also https://docs.rs/slint/latest/slint/docs/mcu/index.html

**Comment by abergmeier** [comment]
Thanks. Sadly I cannot get slint to compile with esp. Is https://github.com/esp-rs/esp-hal/issues/1434 about to be fixed in slint?
